api/v1/sriovoperatorconfig_types.go

```diff
@@ -65,6 +65,8 @@ type SriovOperatorConfigSpec struct {
 	DisablePlugins PluginNameSlice `json:"disablePlugins,omitempty"`
 	// FeatureGates to enable experimental features
 	FeatureGates map[string]bool `json:"featureGates,omitempty"`
+	// ConfigDaemonEnvVars specifies a set of environment variables to be passed to the sriov-network-config-daemon.
+	ConfigDaemonEnvVars map[string]string `json:"configDaemonEnvVars,omitempty"`
```

Comment:
maybe make this one [[]v1.EnvVar](https://github.com/kubernetes/api/blob/fc8a03c10db36574cd45fefb4bb6832a5db0236a/core/v1/types.go#L2267) ? WDYT ?

Reply:
I like the idea. The downside of using `[]corev1.EnvVar` would be that the go templating mechanism would be much more complex. We can easily range over the array, but indenting the maps isn't easy IMO. I can give it a try for sure

Reply:
Implemented, see https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/733/commits/76a8f95581b0449074941997f371ac0903139837